assets/show-more.js

```diff
@@ -0,0 +1,26 @@
+class ShowMoreButton extends HTMLElement {
+  constructor() {
+    super();
+    this.addEventListener('click', (event) => {
+      if (event.target.nodeName === 'SPAN' && event.target.closest('[id^="Show-More-"]')) {
+        this.expandShowMore(event.target.closest('[id^="Show-More-"]'));
+      }
+      if (event.target.nodeName === 'BUTTON' && event.target.closest('[id^="Show-More-"]')) {
+        this.expandShowMore(event.target.closest('[id^="Show-More-"]'));
+        const nextElementToFocus = event.target.parentNode.parentNode.querySelector(".facets__item-show-more")
+        if (!nextElementToFocus.classList.contains('.hidden')) {
+          nextElementToFocus.querySelector('input').focus()
+        }
+      }
+    });
+  }
+  expandShowMore(showMore) {
```

Comment:
Does showMore need to be passed as param? It appears in our usage, we always pass the same element to this method